### ISSUE TITLE
Build docker image for arm68 or Multi-Arch

### ISSUE THREAD

**Issue by jeidison** [kool-dev/kool#407]

**Comment by fabriciojs** [comment]
@jeidison the PHP images did get built for M1, right? What other images do you currently miss/need to have build for M1?

**Comment by aiprogMach** [comment]
Hi what about NPM image? It's so slow on my M1 machine as it's running AMD64 instead of ARM64

**Comment by fabriciojs** [comment]
@aiprogMach sorry, completely missed this question of yours.

Node images still don't have the arm build - when I tried that a while back, it did help with the performance as expected, but the problem is that some common Node packages still rely on building third parties (i.e node-sass) which won't go smoothly on `arm`.

Still this is a must for sure, will get to work on that soon.

Please let me know if you did figure it out on your side, what solution you did to have Node + Docker to perform on M1.

**Comment by aiprogMach** [comment]
@fabriciojs Hello, No worries and thanks for your effort!
We applied a quick solution by cloning your project locally and then run 
`docker buildx build .  --platform linux/arm64 --tag kooldev/node:16-local-arm64`
We're using this "local" image currently on M1 macs and we saw noticeable performance improvement, and we didn't notice any issue so far, however we're not using much packages so our experience might not be reflective to all possible users.
Hopefully this become supported soon!
If I can help with anything let me know!
